tools/rosmaster/src/rosmaster/main.py

```diff
@@ -71,6 +71,10 @@ def rosmaster_main(argv=sys.argv, stdout=sys.stdout, env=os.environ):
     parser.add_option("-t", "--timeout",
                       dest="timeout",
                       help="override the socket connection timeout (in seconds).", metavar="TIMEOUT")
+    parser.add_option("-d", "--debug",
+                      dest="debug", action="store_true", default=False,
+                      help="print debug information")
```

Comment:
I think the help text (and eventually even the option name) should be more specific to what exactly it is doing, namely enabling logging API calls on the master. Maybe the logging level shouldn't be controlled with the same option. That should be configurable already with the rospy logging config file, no?